packages/ui/components/Avatars/index.tsx

```diff
@@ -1,4 +1,5 @@
 import Avatar from "boring-avatars";
+import { sha256 } from "js-sha256";
```

Comment:
Please use the `crypto` package already included in node.js instead of using an extra package:

```
function generateSHA256Hash(input) {
  return createHash('sha256').update(input).digest('hex');
}
```

Reply:
✅